joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 1000

wheel_left_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_left_joint
    pid: {p: 0.0356, i: 0.0, d: 0.000003}

wheel_right_controller:
    type: effort_controllers/JointVelocityController
    joint: wheel_right_joint
    pid: {p: 0.0356, i: 0.0, d: 0.000003}

